.circleci/config.yml

```diff
@@ -57,7 +53,7 @@ jobs:
           name: Install
           command: |
             source venv/bin/activate
-            pip install -e . --no-deps
+            pip install -e '.[default,test,extra,example,doc]'
```

Comment:
It took me a second to figure out why this was failing... it turns out that `example` is not included as a key in `[project.optional-dependencies]` in pyproject.toml. I can't think of any reason it would be intentionally left out of the project config, so I've pushed it up - that should hopefully make this green on CI!